src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
My concern is that two variants - the antichess family and atomic chess - can easily be instantly decided by a single mistake, even when the ramifications are unclear for many moves.  Other variants are much more forgiving, which I suppose makes gaining Elo trickier in the first place?